index.html

```diff
@@ -477,7 +455,41 @@ <h3>Basic Collaboration Mechanics and Information Flow</h3>
   <p>
   As with several other aspects of the system, the flow process is subject to exceptions that may be determined via settings. By default, any participant may send one document to the primary or anyone else up the line at any time, outside of structural channels; if the document turns out to be worthwhile, the recipient may choose to renew the sender's ability to do this. Another setting determines whether a submission flow document, upon being created and sent up the line, also appears randomly at some other point in the pursuance, or at several points, where they may be pushed up the line as usual; this is intended to speed up the proliferation of time-sensitive news items, as well as to better ensure that good input from the margins get more than one chance for acceptance. Another setting allows for documents to automatically proceed to the next position on the line after some variable amount of time has passed without editing or approval/disapproval; this is largely intended to prevent bottlenecking due to temporary absences. Other settings will allow for additional refinement within a pursuance's existing structure.
   </p>
+  
+  <div class="contact">
+    <h1 id="contact">Contact</h1>
+    <hr />
 
+    <h4>General Contact</h4>
+
+    <p>
+      <strong>Email:</strong> <a href="mailto:team@pursuanceproject.org">team@pursuanceproject.org</a>
+      <br />
+      <strong>Twitter:</strong> <a href="https://twitter.com/PursuanceProj">@PursuanceProj</a>
+      <br />
+      <strong>Reddit:</strong> <a href="https://www.reddit.com/r/PursuanceProject/">/r/PursuanceProject</a>
+      <br />
+      <strong>Keybase:</strong> <a href="https://keybase.io/Pursuance">Pursuance</a>
+    </p>
+
+    <h4>Press Contact</h4>
+
+    <p>
+      <strong>Email:</strong> <a href="mailto:press@pursuanceproject.org">press@pursuanceproject.org</a>
+    </p>
```

Comment:
@seabassgonzalez Please undo these whitespace changes so that this PR only shows the changes you made :+1: